glidex.forms.sample/Forms/ImageCellPage.xaml.cs

```diff
@@ -3,13 +3,13 @@
 
 namespace Android.Glide.Sample
 {
-	public partial class ImageCellPage
-	{
-		public ImageCellPage ()
-		{
-			InitializeComponent ();
+    public partial class ImageCellPage
+    {
+        public ImageCellPage()
+        {
+            InitializeComponent();
 
-			BindingContext = Images.RandomSources ().ToArray ();
-		}
-	}
+            BindingContext = Images.RandomSources().ToArray();
+        }
+    }
```

Comment:
Is this just formatting? Why change it?

I will add a `.editorconfig` to help with this in the future.

Reply:
ctrl+E+D shortcun. Sorry for that